### DISCUSSION TITLE
Proposal: auto-property with ref getter

### DISCUSSION THREAD

**Discussion by BreyerW** [dotnet/csharplang#8728] [Language Ideas]
Currently it is impossible to do the following:

```
public ref int intTest{
      ref get;
}=1;
```
It's small weirdo but for consistency with existing auto-property syntax it should be allowed

**Comment by svick** [comment]
I'm not sure consistency is a good enough reason. Especially considering that `ref` properties are fairly rare and that the current syntax is not much more verbose:

```c#
int intTest = 1;
public ref int IntTest { get => ref intTest; }
```
